beetsplug/embedart.py

```diff
@@ -27,6 +26,12 @@
 from beets.util import syspath, normpath, displayable_path
 from beets.util.artresizer import ArtResizer
 from beets import config
+from beets.util.functemplate import Template
+
+__item_template = Template(ui._pick_format(False))
+fmt_item = lambda item: item.evaluate_template(__item_template)
+__album_template = Template(ui._pick_format(True))
+fmt_album = lambda item: item.evaluate_template(__album_template)
```

Comment:
You're right that it's a little unfortunate that we're calling `ui._pick_format` on module initialization. That kind of thing can get brittle. Let's execute your idea about object formatting Real Soon Now™.